tools/topology/topology2/include/pipelines/cavs/passthrough-wov.conf

```diff
@@ -0,0 +1,144 @@
+#
+# CAVS passthrough capture pipeline
+#
+# A simple passthrough pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.passthrough-capture.N.attribute_name".
+#
+# Usage: passthrough-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.passthrough-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/micsel.conf>
+<include/components/dummywov.conf>
+<include/components/pipeline.conf>
+<include/controls/bytes.conf>
+<common_definitions.conf>
+
+Class.Pipeline."passthrough-wov" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!mandatory [
+			"format"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# passthrough-capture objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+
+		virtual."virtual.detect_sink" {
+			type out_drv
+		}
+
+		micsel."1" {
+			Object.Base.audio_format.1 {
+				in_rate		16000
+				out_rate	16000
+				in_channels	2
+				out_channels	1
+				in_ch_map	CHANNEL_MAP_STEREO
+				in_ch_cfg	1
+				out_ch_map	CHANNEL_MAP_MONO
+				out_ch_cfg	0
+				out_bit_depth	32
+				in_bit_depth	32
+				in_valid_bit_depth	32
+				out_valid_bit_depth	32
+			}
+		}
+
+		dummywov."1" {
+			Object.Base.audio_format.1 {
+				in_rate		16000
+				out_rate	16000
+				in_channels	1
+				out_channels	1
+				in_ch_cfg	0
+				in_ch_map	CHANNEL_MAP_MONO
+				out_bit_depth	32
+				in_bit_depth	32
+				in_valid_bit_depth	32
+				out_valid_bit_depth	32
+				ibs			0x80
+				obs			0x80
+			}
+			Object.Control.bytes."1" {
```

Comment:
@libinyang we want to keep this pipeline to be generic enough so that it can be reused with a different wov detection component. In the case of the detect component, it is highly unlikely that we will change the init payload, so please move this entire byte control object within the widget conf file.

Reply:
ok